plugins/tileacq.py

```diff
@@ -478,7 +478,9 @@ def _cancel_acquisition(self, future):
             future.running_subf.cancel()
             logging.debug("Acquisition cancelled.")
 
-        return True
+        dlg.resumeSettings()
```

Comment:
Please don't put this in the canceller. It was fine like this. You need to change the acquisition code, in all the blocks that check for cancel to look like:
```
 if ft._task_state == CANCELLED:
      raise CancelledError("Acquisition cancelled")
```
And at the end, the exception handler would look like:
```         
except CancelledError:
    logging.debug("Acquisition cancelled")
    dlg.resumeSettings()
```